docs/sql-guide/statements/statement-bulk-insert.md

```diff
@@ -0,0 +1,194 @@
+---
+title: BULK INSERT
```

Comment:
Do we need this? we already have https://docs.featurebase.com/docs/sql-guide/statements/statement-insert-bulk/

Reply:
I thought it useful to have a SQL statement in the context of the file rather than getting the user to bounce out to something else.
I'm going to leave it be if that's ok. Although I do think the article is a bit long but it's one of those 99 problems considerations -- it's done for now and I'll come back and take another look later.

Reply:
@ljsinclair You want to have two duplicate pages? this one and https://docs.featurebase.com/docs/sql-guide/statements/statement-insert-bulk/?

Reply:
0_0 hangon. I think I've misunderstood. One moment while I look at the branch

Reply:
Oh #$#*(&$##@!! Two copies of the same file. I am an idiot. Fixing now...

Reply:
Ok. fixed it.
How much not to mention this to anyone? 0_0